tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "strict": true,
     "target": "es2020",
     "module": "commonjs",
+    "moduleResolution": "node",
```

Comment:
what's that?

Reply:
It what i found to make `tsc -m ES2020` succeed.
I'm very unfamiliar with TypeScript, so i don't know what it exactly do.
I've just look in the documentation : https://www.typescriptlang.org/docs/handbook/module-resolution.html#module-resolution-strategies so if module is commonjs moduleResolution default is node, else it's classic. By adding this line i force node resolution so build pass.